packages/app/src/runner/index.ts

```diff
@@ -186,6 +187,29 @@ export async function teardown () {
   isTorndown = true
 }
 
+/**
+ * Add a cross origin iframe for cy.origin support
+ */
+export function addCrossOriginIframe (location) {
```

Comment:
original in [runner/src/iframe/iframes.jsx](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R175)